extensions/adobe/experience/adcloud/experienceevent-all.schema.json

```diff
@@ -35,6 +35,11 @@
           "title": "Ad Asset Details",
           "$ref": "https://ns.adobe.com/experience/adcloud/advertisement",
           "description": "Digital advertisement details"
+        },
+        "https://ns.adobe.com/experience/adcloud/stitchIdChosen": {
```

Comment:
Naming conventions require the following.
"https://ns.adobe.com/experience/adcloud/stitchIDType"
"title": "Stitch Identifier Type"
"description": "Identifier type from the Ad Servers collected through AdCloud STATS to track click-through conversion on browsers that ?clock? third party cookies."

Reply:
Hey @cdegroot-adobe, is this still a concern?